### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by emabrey**
Remove temporary project naming schemes and replace with Tenacity name.

### CODE REVIEW

**Comment by falkTX** [comment]
might be good and take the chance to rename `Mac OS X` to `macOS`

**Comment by adrianheine** [comment]
Is the ancestry still visible in your Readme version?

**Comment by emabrey** [comment]
@adrianheine I'm not sure what you mean. You can look at the changed README [here](https://github.com/tenacityteam/tenacity/blob/c88b24b20e69d1a8d077c59b333ef39191e25c1e/README.md).

File: README.md [hunk 1]
```diff
@@ -32,5 +29,5 @@ Developers may find information for the pre-fork releases available at the [Auda
 
 ## Getting Started
 
-Build instructions for this forked version of Audacity is available [here](BUILDING.md).
+Build instructions for Tenacity are available [here in the BUILDING.md file](BUILDING.md).
```
Comment (Be-ing) [inline README.md]:
```suggestion
Build instructions for Tenacity are available in the [BUILDING.md file](BUILDING.md).
```

**Pull request merged**